src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag

```diff
@@ -0,0 +1,287 @@
+        </c:if>
+        <c:if test="${frc.withLinkToCommentsPage}">
+            <a type="button"
+               href="${frc.linkToCommentsPage}"
+               target="_blank"
+               class="btn btn-default btn-xs icon-button pull-right"
+               data-toggle="tooltip"
+               data-placement="top"
+               title="Edit comment in the Comments page"
+               style="display:none;">
+                <span class="glyphicon glyphicon-new-window glyphicon-primary"></span>
+            </a>
+        </c:if>
+        <c:if test="${frc.editDeleteEnabled}">
+            <form class="responseCommentDeleteForm pull-right">
+                <a href="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_RESPONSE_COMMENT_DELETE %>"
+                   type="button"
+                   id="commentdelete-${divId}"
+                   class="btn btn-default btn-xs icon-button"
+                   data-toggle="tooltip"
+                   data-placement="top"
+                   title="<%= Const.Tooltips.COMMENT_DELETE %>"
+                   <c:if test="${frc.editDeleteEnabledOnlyOnHover}">style="display: none;"</c:if>
+                   <c:if test="${not frc.instructorAllowedToDelete}">disabled="disabled"</c:if>>
+                    <span class="glyphicon glyphicon-trash glyphicon-primary"></span>
+                </a>
+                <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_RESPONSE_ID %>" value="${frc.feedbackResponseId}">
+                <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID %>" value="${frc.commentId}">
+                <input type="hidden" name="<%= Const.ParamsNames.COURSE_ID %>" value="${frc.courseId}">
+                <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>" value="${frc.feedbackSessionName}">
+                <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${frc.googleId}">
+            </form>
+            <a type="button"
+               id="commentedit-${divId}"
+               class="btn btn-default btn-xs icon-button pull-right"
+               onclick="showResponseCommentEditForm(${divIdAsJsParams})"
+               data-toggle="tooltip"
+               data-placement="top"
+               title="<%= Const.Tooltips.COMMENT_EDIT %>"
+               <c:if test="${frc.editDeleteEnabledOnlyOnHover}">style="display: none;"</c:if>
+               <c:if test="${not frc.instructorAllowedToEdit}">disabled="disabled"</c:if>>
+                <span class="glyphicon glyphicon-pencil glyphicon-primary"></span>
+            </a>
+        </c:if>
+    </div>
+    <!-- do not add whitespace between the opening and closing tags -->
+    <div id="plainCommentText-${divId}" style="margin-left: 15px;">${frc.commentText}</div>
+    <c:if test="${frc.editDeleteEnabled}">
+        <form style="display: none;" id="responseCommentEditForm-${divId}" class="responseCommentEditForm">
```

Comment:
Ahha I see, ok then, perhaps we should standardize it some day, but not today, we have a timeline to meet :p